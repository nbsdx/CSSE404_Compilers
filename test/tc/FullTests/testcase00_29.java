/* Invalid: Subclass declared before superclass */
class Foo 
{ 
	public static void main(String[] args) 
	{
	}
}

class Baz extends Bar
{
}

class Bar
{
}