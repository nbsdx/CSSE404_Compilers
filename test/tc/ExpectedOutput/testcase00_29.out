Superclass name Bar not in scope.
